src/lib/components/error-pages/ErrorPage401.component.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import React from 'react';
+import {
+  ErrorPageContainer,
+  Row,
+  WarningIcon,
+  Title,
+  Description,
+  DescriptionContent,
+  Link,
+} from './ErrorPageStyle.jsx';
+import Button from '../buttonv2/Buttonv2.component';
+
+const translations = {
+  en: {
+    unexpected_error: 'Not authorized',
+    error_desc: `You don't have permission to view this page using the credentials that you supplied.`,
+    may_also_contact: 'You may contact ',
+    to_report_issue: ' to report this issue.',
+    return_home: 'Return Home',
+  },
+  fr: {
+    unexpected_error: `Vous n'êtes pas autorisé`,
+    error_desc: `Vous n'êtes pas autorisé à afficher cette page en utilisant les informations d'identification que vous avez fournies.`,
+    may_also_contact: 'Vous pouvez contacter le ',
+    to_report_issue: ' pour signaler ce problème.',
+    return_home: "Retour à l'accueil",
+  },
+};
```

Comment:
In my opinion the translation should be dealt with in the app and not in the design-system.

Reply:
I agree that we should keep the translations within the UIs. 
However, for the error pages, we'd like to reuse the component as much as possible. 
Otherwise,  we will end up duplicating the same translations in all the UIs.